Update csrankings-9.csv

nmattei opened the pull request:
added Tulane University faculty Allan Ding.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-1.csv").

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[0-9].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[0-9].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[0-9].csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

nmattei committed:
Update csrankings-9.csv

added tulane faculty Allan Ding.

emeryberger commented on csrankings-9.csv:
```diff
@@ -2030,6 +2030,7 @@ Zhenjiang Li,City University of Hong Kong,https://www.cs.cityu.edu.hk/~zhenjili/
 Zhenkai Liang,National University of Singapore,https://www.comp.nus.edu.sg/~liangzk,SXfOJjcAAAAJ
 Zhenlin Wang,Michigan Technological University,https://www.mtu.edu/cs/department/faculty-staff/faculty/z-wang,ZAtffHkAAAAJ
 Zhenman Fang,Simon Fraser University,http://www.sfu.ca/~zhenman,v_FAgbgAAAAJ
+Zhengming (Allan) Ding,Tulane Universiry,https://sse.tulane.edu/cs/faculty/Ding,TKbyRRsAAAAJ,
```
Doesn't match DBLP name (https://dblp.org/pid/122/3547.html), has a spurious trailing comma, and "University" is misspelled.

nmattei committed:
Update csrankings-9.csv

Fixed comma, fixed spelling of university, made name consistent with DBLP entry: https://dblp.org/pid/122/3547.html

nmattei commented:
Comments resolved in new update
